### DISCUSSION TITLE
what is: N: Download is performed unsandboxed as root as file '/home/maart/downloads/pipedal_1.2.34_arm64.deb' couldn't be accessed by user '_apt'. - pkgAcquire::Run (13: Permission denied)

### DISCUSSION THREAD

**Discussion by maarthome** [rerdavies/pipedal#159] [Q&A]
Hi Robin,

I wonder... I always get this message... although the installation does not seem to suffer from it:
N: Download is performed unsandboxed as root as file '/home/maart/downloads/pipedal_1.2.34_arm64.deb' couldn't be accessed by user '_apt'. - pkgAcquire::Run (13: Permission denied)

It does not matter whether I am a normal user or root during install...

Cheers

**Comment by rerdavies** [comment]
I don't get the message. I'm using 

      sudo dpkg -i pipedal_1.2.34_arm64.deb 

How are you doing the install?

What are the permissions on `/home/maart/downloads/pipedal_1.2.34_arm64.deb`

**Reply by maarthome** [reply]
I am using what you suggested in your documentation:
sudo apt-get install ./pipedal_1.2.34_arm64.deb

But with 'sudo dpkg -i pipedal_1.2.34_arm64.deb' it does not give a message 😊

Thanks!

PS the IR files loading still is not OK... after selecting the file, it give an OK symbol in front of the file but in fact it is not saved:
journalctl gives:

'Aug 10 22:11:55 pipedal pipedald[1140]: Upload failed. Can't create file "/var/pipedal/audio_uploads/CabIR/Bogie412CB65_CG_C12_0in_Cap.wav".'

Thanks!
Maarten

**Reply by guysoft** [reply]
Same here, should the documentation be changed?

**Comment by rerdavies** [comment]
The documentation has been reverted to use apt-get instead of dpkg.
